content/en/docs/setup/production-environment/tools/kubeadm/high-availability.md

```diff
@@ -17,7 +17,11 @@ and control plane nodes are co-located.
 control plane nodes and etcd members are separated.
 
 Before proceeding, you should carefully consider which approach best meets the needs of your applications
-and environment. [This comparison topic](/docs/setup/independent/ha-topology/) outlines the advantages and disadvantages of each.
+<<<<<<< HEAD:content/en/docs/setup/production-environment/tools/independent/high-availability.md
```

Comment:
This looks like a merge left in by mistake.

Reply:
Thank you it was a miss. Fixed it now.